projects/1.19/assets/miskatonic-mysteries/miskatonicmysteries/lang/zh_cn.json

```
@@ -0,0 +1,507 @@
+	"event.miskatonicmysteries.hastur_party.waiting": "每当夜幕降临，一场黄金聚会便会开始！",
+
+	"item.minecraft.lingering_potion.effect.resonance": "滞留型共鸣药水",
+	"item.minecraft.lingering_potion.effect.resonance_long": "滞留型共鸣药水",
+	"item.minecraft.lingering_potion.effect.resonance_strong": "滞留型共鸣药水",
+	"item.minecraft.potion.effect.resonance": "共鸣药水",
+	"item.minecraft.potion.effect.resonance_long": "共鸣药水",
+	"item.minecraft.potion.effect.resonance_strong": "共鸣药水",
+	"item.minecraft.splash_potion.effect.resonance": "喷溅型共鸣药水",
+	"item.minecraft.splash_potion.effect.resonance_long": "喷溅型共鸣药水",
+	"item.minecraft.splash_potion.effect.resonance_strong": "喷溅型共鸣药水",
+	"item.minecraft.tipped_arrow.effect.resonance": "共鸣之箭",
+	"item.minecraft.tipped_arrow.effect.resonance_long": "共鸣之箭",
+	"item.minecraft.tipped_arrow.effect.resonance_strong": "共鸣之箭",
+	"item.miskatonicmysteries.absinthe": "苦艾酒",
+	"item.miskatonicmysteries.absinthe.tooltip": "附带幻灵之力",
+	"item.miskatonicmysteries.aberration_spawn_egg": "失常者刷怪蛋",
+	"item.miskatonicmysteries.blotter": "吸食纸",
+	"item.miskatonicmysteries.bullet": "枪弹",
+	"item.miskatonicmysteries.byakhee_spawn_egg": "拜亚基刷怪蛋",
+	"item.miskatonicmysteries.chemical_fuel": "化学燃料",
+	"item.miskatonicmysteries.cirrhosus_flesh": "卷曲的肉块",
+	"item.miskatonicmysteries.dark_hood": "暗色头巾",
+	"item.miskatonicmysteries.dark_robe": "暗色长袍",
+	"item.miskatonicmysteries.dark_skirt": "暗色围裙",
+	"item.miskatonicmysteries.elegant_mask": "典雅面具",
+	"item.miskatonicmysteries.enchanted_canvas": "附魔画",
+	"item.miskatonicmysteries.feaster_spawn_egg": "欢宴者刷怪蛋",
+	"item.miskatonicmysteries.feral_mask": "野蛮面具",
+	"item.miskatonicmysteries.fishy_dagger": "散发着腥味的匕首",
+	"item.miskatonicmysteries.fishy_dagger.tooltip": "§2这看起来像是被感染了吗?",
+	"item.miskatonicmysteries.gutting_dagger": "掏空者匕首",
+	"item.miskatonicmysteries.gutting_dagger.tooltip": "§7附加效果：§c失血",
+	"item.miskatonicmysteries.harrow_spawn_egg": "恼灵刷怪蛋",
+	"item.miskatonicmysteries.hastur_bell": "卡尔克萨之钟",
+	"item.miskatonicmysteries.hastur_bell.tooltip": "召唤附近的§e信徒§7和§e王子",
+	"item.miskatonicmysteries.hastur_bell.tooltip.alternate": "Haha bogus",
+	"item.miskatonicmysteries.hastur_cultist_spawn_egg": "黄衣信徒刷怪蛋",
+	"item.miskatonicmysteries.horned_dagger": "长角匕首",
+	"item.miskatonicmysteries.horned_dagger.tooltip": "§7来自黑山羊的致命武器",
+	"item.miskatonicmysteries.incantation_yog": "门和钥匙的符咒",
+	"item.miskatonicmysteries.infested_wheat": "被感染的小麦",
+	"item.miskatonicmysteries.iridescent_pearl": "闪烁的珍珠",
+	"item.miskatonicmysteries.laudanum": "鸦片酊",
+	"item.miskatonicmysteries.necronomicon": "死灵之书",
+	"item.miskatonicmysteries.oceanic_gold": "海洋黄金",
+	"item.miskatonicmysteries.ornate_dagger": "华丽匕首",
+	"item.miskatonicmysteries.ornate_dagger.tooltip": "§7一定概率附加效果：狂躁",
+	"item.miskatonicmysteries.phantasma_spawn_egg": "幻影刷怪蛋",
+	"item.miskatonicmysteries.potent_blotter": "烈性吸食纸",
+	"item.miskatonicmysteries.protagonist_spawn_egg": "研究员刷怪蛋",
+	"item.miskatonicmysteries.re_agent_syringe": "ReAgent™",
```

```suggestion
	"item.miskatonicmysteries.re_agent_syringe": "试剂™",
```
还出了另一个翻译`基金会`但这里是物品...

---

reagent直译确实是试剂
不过这里原文是“ReAgent”，明显“R”和”A“故意大写了，那么其意思肯定不只是“试剂”那么简单。我看了看后面有“™”商标符号，估计这里的“ReAgent”是个（作者生造的）专有名词，，表示这是一个“商标”。所以我保留“ReAgent™”原文，暂不翻译

---

这东西在游戏里面确实是一种试剂，装在注射器里面的

---

emm那就不翻了吧